executions/graphql-kotlin-dataloader-instrumentation/src/test/kotlin/com/expediagroup/graphql/dataloader/instrumentation/fixture/datafetcher/MissionService.kt

```diff
@@ -22,13 +22,16 @@ import com.expediagroup.graphql.dataloader.instrumentation.fixture.extensions.to
 import com.expediagroup.graphql.dataloader.instrumentation.fixture.repository.MissionRepository
 import graphql.schema.DataFetchingEnvironment
 import org.dataloader.BatchLoader
+import org.dataloader.DataLoaderOptions
+import org.dataloader.stats.SimpleStatisticsCollector
 import java.util.Optional
 import java.util.concurrent.CompletableFuture
 
 data class MissionServiceRequest(val id: Int, val astronautId: Int = -1)
 
 class MissionDataLoader : KotlinDataLoader<MissionServiceRequest, Mission?> {
     override val dataLoaderName: String = "MissionDataLoader"
+    override fun getOptions(): DataLoaderOptions = DataLoaderOptions.newOptions().setStatisticsCollector { SimpleStatisticsCollector() }
```

Comment:
for testing purposes we need statistics but in this new release, those are now NOOP by default